dpv/README.md

```diff
@@ -0,0 +1,33 @@
+# /dpv
+
+This [w3id](https://w3id.org/) provides a persistent URI namespace for Data Privacy Vocabulary (DPV) and other DPVCG resources. The Data Privacy Vocabulary (DPV) provides concepts to describe and represent information about processing of personal data.
+
+## Uses
+
+|-------------------+----------------------------------------+---------------------------------------------|
```

Comment:
I think you either want to use mardown table syntax or just surround the table with backticks so it formats in a readable way.

Reply:
Hi, thanks. I've added the backticks to README.md and force pushed changes so it still remains one commit.